src/app/help/get.controller.js

```diff
@@ -2,5 +2,12 @@ const logger = require('../../common/utils/logger')(__filename);
 
 module.exports = (req, res) => {
   logger.debug('In help get controller');
-  res.render('app/help/index');
+
+  if(req.query.tab && req.query.tab.toUpperCase() === 'UPT' ){
```

Comment:
Question: Was this implemented instead of straight route url for flexibility?
- The above is confusing.